src/Components/RecentJobs/RecentBox/SingleJob/index.jsx

```diff
@@ -0,0 +1,29 @@
+import Skill from "./Skill/Skill"
+
+function SingleJob({logo, JobTitle, company, type, salary, skills}) {
```

Comment:
Can you make the J in JobTitle a little j please, just to be consistent with the other props